src/components/AvatarGroup/avatars.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import StyledAvatar from './styled/avatar';
+
+export default function Avatars(props) {
+    const { avatars, maxAvatars, showCounter, size } = props;
+    const total = avatars.length;
+    const maxAvatar = total > maxAvatars && showCounter ? maxAvatars - 1 : maxAvatars;
```

Comment:
There is no relation between max avatar to show and counter if there are 100 avatars, maxAvatar=5 and showCounter is passed then the component should render 5 avatars plus the counter.